media_mpx.routing.yml

```diff
@@ -37,3 +37,23 @@ media_mpx.settings:
     _title: 'mpx settings'
   requirements:
     _permission: 'administer site configuration'
+
+media_mpx.asset_sync:
+  path: '/admin/config/media/mpx/asset-sync'
```

Comment:
Do you think these should be under config, or a subtask under content/media?

Reply:
That's a good point. Given this is a contrib and we don't know how bloated media can be already, maybe a dedicated area? https://monosnap.com/file/ovTWNU9MUYwUUyYYurEddXzvnxQDRv

I've no hard preference either way. If that's ok I'll tackle the moving of this in the next task, given there are more UI items to come.

Reply:
In D7, it's under Media -> mpx, I think it makes sense as a subtask there? We can always move it later. Either way config is better than config since most editors wouldn't see config sections of admin.